src/model/image-tag.ts

```diff
@@ -35,25 +35,61 @@ class ImageTag {
       webgl: 'webgl',
       mac: 'mac-mono',
       windows: 'windows-mono',
+      windowsIl2cpp: 'windows-il2cpp',
+      wsaplayer: 'universal-windows-platform',
```

Comment:
could you camelcase `wsaPlayer` here please? I'm working on syncing versioning backend with your changes.

Reply:
Sure, should it be WSAPlayer since wsa is an acronym or just wsaPlayer?

Reply:
We use lowercase acronyms in all my projects including all GameCI repos.

See also most accepted answer here https://stackoverflow.com/questions/15526107/acronyms-in-camelcase